### PR TITLE
US-18 Direct Acces

### PR DESCRIPTION
**Pull request by AshBll**

### CODE REVIEW

**Review by FranciscoKevin — DISMISSED** [review]
Good l'ami !

**Review by maxime-auneau — APPROVED** [review]
OK

**Review by maxime-auneau — DISMISSED** [review]
ok

File: src/Model/TroopManager.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        // prepared request
+            $insert = $this->pdo->prepare("INSERT INTO " . self::TABLE . " (name, strength) VALUES (:name, :strength)");
+            $insert->bindValue('name', $troop->getName(), PDO::PARAM_STR);
+            $insert->bindValue('strength', $troop->getLevel(), PDO::PARAM_INT);
+        if ($insert->execute()) {
+            return (int)$this->pdo->lastInsertId();
+        }
+    }
+
+    public function delete(): void
```
Comment (RomainClair) [inline src/Model/TroopManager.php:43]: Wouldn't it be more accurate to call this method deleteAll ?

File: src/View/layout.html.twig [hunk 2]
```diff
@@ -1,22 +1,20 @@
...
 </head>
-
 <body>
-
-{% block content %}{% endblock %}
-{% block javascript %}{% endblock %}
+    {% block header %}{% endblock %}
+    {% block gametable %}{% endblock %}
+    {% block burger %}{% endblock %}
+    {% block content %}{% endblock %}
```
Comment (RomainClair) [inline src/View/layout.html.twig]: Do you really need that amount of consecutive blocks ?

**Review by RomainClair — CHANGES_REQUESTED** [review]
Are you sure you want to change the project dependencies ? (composer.lock was changed)

The terrain (copie).jpg file should probably be renamed or remove

You should have a comment at the beginning of every class and before each method to explain what it is for and how to use it.

**Review by maxime-auneau — APPROVED** [review]

**Review by thocle86 — APPROVED** [review]
Ok

**Review by FranciscoKevin — APPROVED** [review]
C'est good !

**Review by RomainClair — CHANGES_REQUESTED** [review]
Are you sure you want to change the composer.lock ? It means you changed the dependencies of your project. And it seems to be a mistake in that case.

Don't add the same file in 2 differents PR. It's error prone.
And it asks me to review the same code twice...

**Pull request closed without merging**